sections/all_map_locations.tex

```diff
@@ -226,13 +226,11 @@ \subsection*{Towns, Mines and Settlements}
 \columnbreak
 
 \textbf{Settlements}\index{Settlement} act as a spawn point for Secondary Heroes, and as a place for Main Heroes to move to when defeated.
-When you Flag a Settlement, choose whether to increase your \svg{gold}, \svg{building_materials} or \svg{valuablegreater} income by one space.
-As with Mines, if you are the first player to Flag a Settlement, you \textbf{immediately} gain Resources equal to that increase in production.
-Mark the Settlement with an appropriate Resource Token to show which Resource it produces.
-When you Flag an enemy Settlement, you may change this Resource.\par
-Additionally, \textbf{instead of increasing Resource Production}, you may choose to \textbf{Reinforce} one of your \svgunit{bronze} or \svgunit{silver} Units immediately for half the normal cost, rounded up.
-If you were the first player to Flag the Settlement, Reinforce that Unit \textbf{for free} instead.
-Do not place any Resource Tokens on the Settlement if you choose to Reinforce.
+When you Flag a Settlement, choose one of the following options: 
+\begin{itemize} 
+	\item Increase your \svg{gold}, \svg{building_materials} or \svg{valuablegreater} income by one space. As with Mines, if you are the first player to Flag a Settlement, you \textbf{immediately} gain Resources equal to that increase in production. Mark the Settlement with an appropriate Resource Token to show which Resource it produces. When you Flag an enemy Settlement, you may change this Resource.
+	\item \textbf{instead of increasing Resource Production}, you may choose to \textbf{Reinforce} one of your \svgunit{bronze} or \svgunit{silver} Units immediately for half the normal cost, rounded up. If you were the first player to Flag the Settlement, Reinforce that Unit \textbf{for free} instead. Do not place any Resource Tokens on the Settlement if you choose to Reinforce.
```

Comment:
```suggestion
  \item \textbf{instead of increasing Resource Production}, you may choose to \textbf{Reinforce} one of your \svgunit{bronze} or \svgunit{silver} Units immediately for half the normal cost, rounded up.
    If you were the first player to Flag the Settlement, Reinforce that Unit \textbf{for free} instead.
    Do not place any Resource Tokens on the Settlement if you choose to Reinforce.
```